src/main/kotlin/lotto/domain/LottoShop.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain
+
+private const val UNIT = 1000
+
+object LottoShop {
+    fun getNumberOfPurchase(amount: Int): Int {
+        require(amount % UNIT == 0) {
+            throw RuntimeException("$UNIT 단위로 입력해주세요.")
+        }
+        return amount / UNIT
+    }
+
+    private fun generateLottoNumbers(): List<Int> {
+        val allNumbers = (1..45).toList()
+        val lottoNumbers = allNumbers.shuffled().subList(0, 6)
+        return lottoNumbers.sorted()
+    }
+
+    fun getLottoNumbers(count: Int): List<List<Int>> {
+        return List(count) { generateLottoNumbers() }
+    }
```

Comment:
`generateLottoNumbers()`는 `getLottoNumbers(count: Int)`내에서 선언되고 있는것으로 보입니다.

이럴때는 참조당하는 메서드를 아래로 내리는 것이 코드 가독성 향상에 큰 도움이 됩니다.

이 부분 뿐 아니라 다른곳에도 해당되는 부분이 보이니 동일하게 개선해주세요.

https://kotlinlang.org/docs/coding-conventions.html#class-layout
> put related stuff together, so that someone reading the class from top to bottom can follow the logic of what's happening.